test-framework/core/src/main/java/org/keycloak/testframework/admin/AdminClientFactorySupplier.java

```diff
@@ -15,12 +15,12 @@ public class AdminClientFactorySupplier implements Supplier<AdminClientFactory,
     public AdminClientFactory getValue(InstanceContext<AdminClientFactory, InjectAdminClientFactory> instanceContext) {
         KeycloakServer server = instanceContext.getDependency(KeycloakServer.class);
 
-        if (!server.isTlsEnabled()) {
-            return new AdminClientFactory(server.getBaseUrl());
-        } else {
+        if (server.isTlsEnabled() || server.isFipsEnabled() ) {
```

Comment:
I don't like this change - simplest would just be to make the FIPS Suite enable TLS as well (or you can always make the config set TLS when FIPS is enabled).